docs/src/docs/asciidoc/user/management.adoc

```diff
@@ -0,0 +1,63 @@
+include::common.adoc[]
+
+= Management and Monitoring
+
+include::menu.adoc[]
+
+== Intro
+
+Managed objects like caches, cache managers and stores are registered into a `org.ehcache.management.ManagementRegistry`
+instance.
+
+A `ManagementRegistry` implementation has to understand the registered object and provide management and monitoring
+capabilities for them, including the capabilities' context.
+
+Given a capability and a context, statistics can be collected or calls can be made.
+
+The current `ManagementRegistry` implementation provides minimal support for `Ehcache` instances, providing
+a minimal set of statistics and actions via a couple of capabilities.
+
+
+== Making use of the `ManagementRegistry`
+
+By default, a `ManagementRegistry` is automatically discovered and enabled, but can only be accessed by ehcache
+internal services. If you wish to make use of it, you should create your own instance and pass it to the
+cache manager builder as a service:
+
+[source,java]
+----
+ManagementRegistry managementRegistry = new DefaultManagementRegistry(); //// <1>
```

Comment:
Can this be moved to be real sample code?
Ideally the code lives in a class in `org.ehcache.docs` package in one of the modules, which enables specific check style rules on it.
The 4 slashes are not needed, only two. Check other samples. The output includes the number as an image that is not copied over when you copy the code.